src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $totalSize + $totalGap + 10px;
+  padding: 10px;
+
+  $days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} .calendar__day:first-child {
+      margin-left: ($number * ($daySize + $dayGap)) + 3 * $dayGap;
+    }
+  }
+
+  @for $lastday from 28 through 31 {
+    &--month-length--#{$lastday} .calendar__day:nth-child(n + #{$lastday + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    height: $daySize;
+    width: $daySize;
+    font-family: $BodyFont;
+    size: $daySize;
+    gap: $dayGap;
```

Comment:
Do these props really work?
You have to use `width`, `height`, `margin-right` and `margin-botton` properties instead of these.
For example:
```
width: $daySize;
height: $daySize;
margin-right: $dayGap;
margin-bottom: $dayGap;
```